textpipe/doc.py

```diff
@@ -274,3 +275,20 @@ def _set_default_nlp(self, lang):
             self._spacy_nlps[lang] = {}
         model = spacy.load('{}_core_{}_sm'.format(lang, 'web' if lang == 'en' else 'news'))
         self._spacy_nlps[lang][None] = model
+
+    @functools.lru_cache()
+    def sentiment(self):
```

Comment:
this can now be a property

Reply:
Does not seem to be working:

Failed example:
    doc.sentiment()
Exception raised:
    Traceback (most recent call last):
      File "/snap/pycharm-community/85/helpers/pycharm/docrunner.py", line 140, in __run
        compileflags, 1), test.globs)
      File "<doctest doc.Doc.sentiment[1]>", line 1, in <module>
        doc.sentiment()
    TypeError: 'Score' object is not callable

I take it that Score is something from pattern

Reply:
you should remove the `()` when 'calling` a property (from the operation, for instance)